CONTRIBUTING.md

```diff
@@ -371,6 +371,13 @@ There are several ways to run the tests. Requirements are:
 - An empty repository for testing gitops operations
 - A private key to that gitops repository
 
+The integration tests require the following to be set:
+```BASH
+export IT_SUBNET_1 # Subnet used to create manual eks cluster
```

Comment:
I could also imagine instead passing in the VPC ID and just taking 2 subnets, could be simpler at least from a configuration point of view.

Reply:
Yeah that would make it simper, but I don't think we currently have anyway of querying the VPC api so we'd have to setup a new client etc so not sure if its worth it. I was thinking about removing the env vars and setting up a CF template to create the network/roles but I don't know if its worth it either 🤷‍♂️

Reply:
yeah it's definitely fine as is

Reply:
i think it's worth using CFN to create the necessary infra though, instead of clicking it together so no one knows the specifics of the prerequisites